### ISSUE TITLE
Inconsistencies between fast-text-field and fast-text-area

### ISSUE THREAD

**Issue by ben-girardet** [microsoft/fast#3688]
**Describe the bug; what happened?**

When using `<fast-text-field>` and `<fast-text-area>` with a `width: 100%;` they do not act in a consistent way.

* fast-text-field grow to 100% but not its control do not fit its entire width
* fast-text-area do not grow to 100% but its control fits its entire width

**What are the steps to reproduce the issue?**

```html
<div class="container">
  <fast-text-field>
    Fast text field
  </fast-text-field>
  <fast-text-area>
    Fast text area
  </fast-text-area>
</div>
```

```css
.container {
  width: 600px;
  margin: auto;
  padding: 40px 0;
}

.container fast-text-field,
.container fast-text-area {
  width: 100%;
  border: dotted 2px #fcc;
}

.container fast-text-field::part(control),
.container fast-text-area::part(control) {
  background-color: #ccc; /* in order to reveal the control width bug */
} 
```

**What behavior did you expect?**

The text area should take the whole width and the control inside each field must take the whole width of the field itself

**If applicable, provide screenshots:**

<img width="621" alt="Capture d’écran 2020-08-11 à 21 23 10" src="https://user-images.githubusercontent.com/4275071/89939945-10100580-dc19-11ea-87ff-67b87bb5b2bc.png">

**In what environment did you see the issue?**

* MacOS
* Chrome

**Comment by ben-girardet** [comment]
Probably related, when using a slotted content in the `start` slot it pushes the control away to the right.

```html
<div class="container">
  <fast-text-field>
    <span slot="start"></span> <!-- faking some start slot content -->
    Fast text field
  </fast-text-field>
</div>
```

```css
.container {
  width: 600px;
  margin: auto;
  padding: 40px 0;
}

.container fast-text-field {
  width: 100%;
  border: dotted 2px #fcc;
}

.container fast-text-field::part(control) {
  background-color: #ccc; /* in order to reveal the control width bug */
} 

.container fast-text-field::part(start) {
  width: 24px;
  height: 24px;
  background: red;
  border-radius: 4px;
}
```

Here is a screenshot of this bug:

<img width="631" alt="Capture d’écran 2020-08-11 à 21 26 57" src="https://user-images.githubusercontent.com/4275071/89940181-68470780-dc19-11ea-8e9d-a97e9455ae1a.png">

**Comment by chrisdholt** [comment]
Thanks @ben-girardet - I *think* we’ve addressed the width issue and that should release this week. I’ll look into the second issue with start content now. Once I find the related PR for the width I’ll link here (on my phone currently).

**Comment by ben-girardet** [comment]
Thanks @chrisdholt for your answer. I've found [this PR which aims to solve the width 100% for text-area](https://github.com/microsoft/fast/pull/3623) but I'm afraid it doesn't address the `start` slot issue. 

Would be great to test the above use cases against the current fixes

**Comment by chrisdholt** [comment]
@ben-girardet Yes, that's the PR and it for sure doesn't address start/end issues. I'm looking into that now and I'll be aiming to get this out for our Thursday release. Thanks for filing this!

**Comment by chrisdholt** [comment]
@ben-girardet For the start problem you are seeing above, what version of Chrome are you running in (and is it the stable channel)? Upon booting this up in our test environment I'm not reproing with the above implementation. Is this in the `@microsoft/fast-components` package and if so - do you know what version?

![image](https://user-images.githubusercontent.com/13071055/90044112-04a4f480-dc82-11ea-867b-99a75e40b6b4.png)

**Comment by chrisdholt** [comment]
Appears as though this repro's in the Fluent UI package, likely because we hadn't don the work to fix the 100% width scenario there yet. A PR should be up today to fix it there as well, and the fix should release overnight 👍

**Comment by ben-girardet** [comment]
Tested on Chrome 84.0.4147.125 and with fast-components@1.7.0

**Comment by chrisdholt** [comment]
Awesome, thanks @ben-girardet! I'm fairly sure that this should be fixed in our next release (This Thursday). I was able to repro this in the Fluent repo which didn't have the fix of #3623. As you can see from the above, I *think* this should be solved for you as the control currently checked-in shows as above, which seems to be what's expected.

**Comment by ben-girardet** [comment]
Ok thanks!! Then we can close this issue - I'll come back here after release in case it needs more attention...
